funnel/views/schedule.py

```diff
@@ -226,7 +244,7 @@ def schedule_json(self):
 
     @route('ical')
     @requires_roles({'reader'})
-    def schedule_ical(self):
+    def schedule_ical_download(self):
```

Comment:
Why is this a dupe of the function above? One function can call the other function.